docs/operation/cluster_and_service_configuration.rst

```diff
@@ -324,10 +324,41 @@ This functionality can be actived:
    than `retention_size`. You should at least add a 10% margin to be safe.
    (i.e.: set `retention_size` to 9GB for a 10GB volume)
 
-Both size and time based retentions can be activated at the same time.
+Then :ref:`apply the configuration<csc-prometheus-apply-cfg>`.
+
+Set Kubelet metrics scrape tiemout
+""""""""""""""""""""""""""""""""""
+
+When using a bunch of sparseloopdevice kubelet metrics endpoint may spend
+a lot of time to retrieve volumes metrics. In such situation the default
+10s scrape timeout of Prometheus is not enough as the metrics endpoint may
+take longer to answer.
+
+To avoid losing data in such situations, we enabled customization of kubelet
+scarpe timeout, to customize it use the following configuration:
```

Comment:
```suggestion
In some cases (e.g. when using a lot of sparse loop devices), the kubelet metrics
endpoint can be very slow to answer and the Prometheus' default 10s scrape
timeout may not sufficient. 

To avoid timeouts and thus losing metrics, you can customize the scrape timeout as follows:
```